src/components/Tree/TreeTitle.tsx

```diff
@@ -0,0 +1,72 @@
+import * as _ from 'lodash'
+import * as PropTypes from 'prop-types'
+import * as React from 'react'
+
+import { UIComponent, childrenExist, createShorthandFactory } from '../../lib'
+import { treeTitleBehavior } from '../../lib/accessibility'
+import { Accessibility } from '../../lib/accessibility/types'
+import {
+  commonUIComponentPropTypes,
+  childrenComponentPropTypes,
+  contentComponentPropsTypes,
+} from '../../lib/commonPropTypes'
+import {
+  UIComponentProps,
+  ChildrenComponentProps,
+  ContentComponentProps,
+} from '../../lib/commonPropInterfaces'
+
+export interface TreeTitleProps
+  extends UIComponentProps<any, any>,
+    ChildrenComponentProps,
+    ContentComponentProps {
+  /** Whether or not the subtree of the item is in the open state. */
+  active?: boolean
+
+  /** Whether or not the item has a subtree. */
+  hasSubtree?: boolean
+}
+
+class TreeTitle extends UIComponent<TreeTitleProps, any> {
+  static create: Function
+
+  static className = 'tree-list__title'
```

Comment:
Please prefix with `ui-`:

```suggestion
  static className = 'ui-tree-list__title'
```

Other our components do the same:
https://github.com/stardust-ui/react/blob/master/src/components/Button/Button.tsx#L106
https://github.com/stardust-ui/react/blob/master/src/components/Header/HeaderDescription.tsx#L27